README.md

```diff
@@ -64,14 +64,18 @@ To determine whether to allow response _response_ to a request _request_, run th
 1. If _mimeType_ is not failure, then:
    1. If _mimeType_ is an opaque-safelisted MIME type, then return true.
    1. If _mimeType_ is an opaque-blocklisted-never-sniffed MIME type, then return false.
-   1. If _response_'s status is `206` and _mimeType_ is an opaque-blocklisted MIME type, then return false. TODO: is this needed with the requesters set?
+   1. If _response_'s status is 206 and _mimeType_ is an opaque-blocklisted MIME type, then return false.
    1. If _nosniff_ is true and _mimeType_ is an opaque-blocklisted MIME type or its essence is "`text/plain`", then return false.
-1. If the user agent's opaque-safelisted requesters set contains (_request_'s opaque media identifier, _request_'s current URL), then return true.
+1. If the user agent's opaque-safelisted media URLs set contains (_request_'s opaque media identifier, _request_'s current URL), then return true.
 1. Wait for 1024 bytes of _response_ or end-of-file, whichever comes first and let _bytes_ be those bytes.
-1. If the [image type pattern matching algorithm](https://mimesniff.spec.whatwg.org/#image-type-pattern-matching-algorithm) given _bytes_ does not return undefined, then return true.
 1. If the [audio or video type pattern matching algorithm](https://mimesniff.spec.whatwg.org/#audio-or-video-type-pattern-matching-algorithm) given _bytes_ does not return undefined, then:
-   1. Append (_request_'s opaque media identifier, _request_'s current URL) to the user agent's opaque-safelisted requesters set.
-   1. Return true.
+   1. If _requests_'s opaque media identifier is null, then return false.
+   1. If _response_'s status is not an [ok status](https://fetch.spec.whatwg.org/#ok-status), then return false.
```

Comment:
It seems that Chrome might be stricter here and require 200 or 206 (and not allow 299). In Safari I could not even get 200 to work. It does a request for the first byte and it seems to abort if the server does not do range requests?

cc @youennf